vtex.storefront-permissions/configuration.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "Organization",
```

Comment:
When I was testing permissions for orderquote, I had to set the `name` as `vtex.orderquote` in order to get the correct permissions back from the backend `checkUserPermissions` GraphQL query. When the query executes, doesn't it look at the app ID of the sender (without the version number) and then return any features matching that ID?

Reply:
Also, I think this permissions configuration and builder should be moved to the frontend app, right?